pxr/usdImaging/lib/usdImaging/delegate.cpp

```diff
@@ -1582,6 +1588,22 @@ UsdImagingDelegate::SetWindowPolicy(CameraUtilConformWindowPolicy policy)
     }
 }
 
+void
+UsdImagingDelegate::SetDisplayUnloadedPrimsWithBounds(bool displayUnloaded)
```

Comment:
A question for spiff, do we need to check _enableUsdDrawModes here?

Reply:
I _think_ it's fine to leave them decoupled... the doc for this method states it won't be effective if draw modes are not disabled, and by leaving them decoupled, the client can configure them in either order without frustration.